docs/modules/ROOT/pages/documentation/payara-server/server-configuration/certificate-management.adoc

```diff
@@ -106,6 +106,79 @@ asadmin> generate-self-signed-certificate --dn "CN=test.payara.fish,IP=192.168.1
 test_cert
 ----
 
+[[renew-self-signed-certificates]]
+== `renew-self-signed-certificates`
+
+Usage::
+`asadmin> renew-self-signed-certificates
+
+Aim::
+The renew-self-signed-certificates subcommand renews a self-certificate using the old distinguished name and alias.
```

Comment:
```suggestion
The `renew-self-signed-certificates` subcommand renews a self-signed certificate using the old distinguished name and alias used when it was previously generated using the `generate-self-signed-certificate` command.
```

Reply:
Does the command renew a single certificate or all self-signed certificates in the keystore? Sometimes the documentation suggests it renews a single certificate and sometimes it suggests it will renew all certificates. 

If it renews a single certificate, how it's specified? I'm missing a way to specify the certificate alias.

If it renews all certificates, does it renew all of them or only those that are close to the expiration date? In this case, can we add an option to allow specifying a single alias to renew?